eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Internal.hs

```diff
@@ -434,11 +436,18 @@ votingCommitteeThresholdInternal pp committee = \case
     threshold =
       case committeeThreshold <$> committee of
         -- if the committee size is smaller than the mnimimum given in pparams,
```

Comment:
Another typo in the edited comment: `s/mnimimum/minimum`

Reply:
Thanks!